src/stores/ProjectStore.ts

```diff
@@ -20,6 +20,8 @@ import { SegmentFeatureStore } from './SegmentFeatureStore'
 import { ProjectImportStore } from './ProjectImportStore'
 import { Coordinate } from '../interfaces/ImageInterfaces'
 import { reverseTransform } from '../lib/plot/Helper'
+import { PlotStatistic } from '../definitions/UIDefinitions'
+import { select } from 'underscore'
```

Comment:
You can remove this. Autocomplete/import will occasionally import this if you're typing select and accidentally press enter.